Sources/Sound.swift

```diff
@@ -131,6 +131,7 @@ open class Sound {
         for _ in 0..<playersPerSound {
             do {
                 let player = try Sound.playerClass.init(contentsOf: url)
+                player.enableRate = true
```

Comment:
why do you set `enableRate` to `true` by default? I'm guessing that `AVAudioPlayer` doesn't do it to improve performance